### PR TITLE
Add clustering

### PR DESCRIPTION
**Pull request by richard-still-ft**
Should yield a big boost in performance or the ability to significantly reduce costs. 

** Needs a new env var to be set `WEB_MEMORY` Should be 1.7Gb.

### CODE REVIEW

File: server/cluster.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const cluster = require('cluster');
+const control = require('strong-cluster-control');
+const startApp = require('./api.js');
+const clusterSize = process.env.CLUSTER_SIZE || 1;
```
Comment (commuterjoy) [inline server/cluster.js]: `parseInt(process.env.CLUSTER_SIZE)` ?

File: server/cluster.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const cluster = require('cluster');
+const control = require('strong-cluster-control');
+const startApp = require('./api.js');
+const clusterSize = process.env.WEB_CONCURRENCY || 4;
```
Comment (commuterjoy) [inline server/cluster.js]: `parseInt(process.env.WEB_CONCURRENCY)` ?

**Comment by commuterjoy** [comment]
`CLUSTER_SIZE` will need setting on the heroku box directly

File: server/cluster.js (hunk 2 above)
Comment (richard-still-ft) [inline server/cluster.js]: Not been needed on other apps thus far. Switched to `WEB_CONCURRENCY ` to be used in conjunction with `WEB_MEMORY`

Comment (richard-still-ft) [inline server/cluster.js]: So it turns out `WEB_CONCURRENCY` is not set because of the way we deploy via Haikro. So will set it manually

**Comment by commuterjoy** [comment]
+1 please report back on stats :)

**Pull request merged**